openslides_backend/action/mediafile/set_as_logo.py

```diff
@@ -0,0 +1,49 @@
+from typing import Any, Dict
+
+from ...models.models import Mediafile
+from ...shared.exceptions import ActionException
+from ...shared.patterns import FullQualifiedId
+from ...shared.schema import schema_version
+from ..generics import UpdateAction
+from ..register import register_action
+
+
+@register_action("mediafile.set_as_logo")
+class MediafileSetAsLogoAction(UpdateAction):
+    """
+    Action to set a mediafile as logo.
+    """
+
+    model = Mediafile()
+    schema = {
```

Comment:
Please use the default schema. It should never be needed to explicitly define a schema

Reply:
Done.